runtime/calamari/src/migrations/asset_id.rs

```diff
@@ -104,6 +104,21 @@ pub struct OldAssetRegistrarMetadata {
     pub is_sufficient: bool,
 }
 
```

Comment:
Throwing out that we could not keep maintaining this as it already was migrated and shouldn't be needed again.

Reply:
right, but i feel like it's a good idea to keep db migration history .
What if we just comment out the modules of the unused ones in mod.rs ?